pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -617,9 +617,17 @@ func parseAppInstanceConfig(config *zconfig.EdgeDevConfig,
 		for _, adapter := range cfgApp.Adapters {
 			log.Tracef("Processing adapter type %d name %s",
 				adapter.Type, adapter.Name)
-			appInstance.IoAdapterList = append(appInstance.IoAdapterList,
-				types.IoAdapter{Type: types.IoType(adapter.Type),
-					Name: adapter.Name})
+			hwaddr, err := net.ParseMAC(adapter.EthVf.Mac)
```

Comment:
This gets a segmentation violation; presumable adapter.EthVf is nil.

Reply:
Presumably this belongs in the iPhyIoNetEthVF branch below. But is it always the case that the model needs to specify a MAC address or can the PF pick one if none specified?

Reply:
If MAC address is not specified by user, it'll be specified by driver in VM. NIC driver can only specify PCI address of the device (and in most cases, it should)

Reply:
Ok